File: sdk/clojure/src/hitechcloud/sdk/duoc_pham_and_y_te.clj
(ns hitechcloud.sdk.duoc_pham_and_y_te
  (:require [hitechcloud.sdk.http-client :as http]))

;; Dược phẩm & Y tế

(defn health_drug_prices "GET /api/health/drug/prices - Tra các lượt kê khai giá bán buôn thuốc dự kiế..." [client & [params]] (http-get client "/api/health/drug/prices" params))

(defn health_drug_tenders "GET /api/health/drug/tenders - Tra kết quả trúng thầu thuốc tại các cơ sở y t..." [client & [params]] (http-get client "/api/health/drug/tenders" params))

(defn health_drugs "GET /api/health/drugs - Tìm trong sổ đăng ký thuốc được phép lưu hành..." [client & [params]] (http-get client "/api/health/drugs" params))

(defn health_drugs_2 "GET /api/health/drugs/2 - Hồ sơ đầy đủ của MỘT SỐ ĐĂNG KÝ" [client & [params]] (http-get client "/api/health/drugs/2" params))

(defn health_status "GET /api/health/status - Độ tươi của ba bản chụp trên máy chủ này: số b..." [client & [params]] (http-get client "/api/health/status" params))

File: sdk/fsharp/src/HiTechCloudSDK/DuocPhamAndYTe.fs
namespace HiTechCloud.SDK

/// Dược phẩm & Y tế
type DuocPhamAndYTeResource(httpClient: HttpClient) =

    /// GET /api/health/drug/prices - Tra các lượt kê khai giá bán buôn thuốc dự kiế...
    member this.HealthDrugPrices() =
        async {
            return! httpClient.getAsync("/api/health/drug/prices")
        }

    /// GET /api/health/drug/tenders - Tra kết quả trúng thầu thuốc tại các cơ sở y t...
    member this.HealthDrugTenders() =
        async {
            return! httpClient.getAsync("/api/health/drug/tenders")
        }

    /// GET /api/health/drugs - Tìm trong sổ đăng ký thuốc được phép lưu hành...
    member this.HealthDrugs() =
        async {
            return! httpClient.getAsync("/api/health/drugs")
        }

    /// GET /api/health/drugs/2 - Hồ sơ đầy đủ của MỘT SỐ ĐĂNG KÝ
    member this.HealthDrugs2() =
        async {
            return! httpClient.getAsync("/api/health/drugs/2")
        }

    /// GET /api/health/status - Độ tươi của ba bản chụp trên máy chủ này: số b...
    member this.HealthStatus() =
        async {
            return! httpClient.getAsync("/api/health/status")
        }

